src/main/resources/swagger.api/schemas/entityTypeDefaultSort.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Default sort criteria for an entity type",
+  "description": "A definition of default sort criteria for an entity type",
+  "type": "object",
+  "properties": {
+    "columnName": {
```

Comment:
Entity doesn't have columns, it has fields. I think it is better to use the fields (from entity) here, not the name of the columns. Please change it to "field"

Reply:
Hi @MikhailFokanov 
As we discussed today, the name "columnName" was chosen to match with the existing "columns" element.